.idea/workspace.xml

```diff
@@ -0,0 +1,725 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this file must not be committed (look at https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore to see how to automatically ignore such files generated by your IDE)

Reply:
lol I usually remove this manually before committing. But thanks for the link for ignoring the files globally!